### PR TITLE
Add PyPy 5.7.1

### PR DESCRIPTION
**Pull request by joshfriend**
https://morepypy.blogspot.com/2017/04/pypy-571-bugfix-released.html

### CODE REVIEW

**Comment by kennethreitz** [comment]
almost but not quite!

File: builds/runtimes/pypy-5.7.1 [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Build Deps: libraries/sqlite
+
+# NOTICE: This formula only works for the cedar-14 stack, not cedar.
+
+OUT_PREFIX=$1
+
+echo "Building PyPy..."
+SOURCE_TARBALL='https://bitbucket.org/pypy/pypy/downloads/pypy2-v5.7.1-linux64.tar.bz2'
+curl -L $SOURCE_TARBALL | tar jx
+cp -R pypy2-v5.7.0-linux64/* $OUT_PREFIX
```
Comment (cclauss) [inline builds/runtimes/pypy-5.7.1]: Line 10 says v5.7.1 and line 12 says v5.7.0.

Perhaps the best (DRY) solution would be to put `PYPY_VERSION="pypy2-v5.7.1"` on line 8 that then use `${PYPY_VERSION}` on lines 9, 10, and 12.

Comment (joshfriend) [inline builds/runtimes/pypy-5.7.1]: Fixed! Sorry, not sure what was wrong with me there 😕

**Comment by kennethreitz** [comment]
Merged.

**Comment by kennethreitz** [comment]
For next time: make sure you `chmod +x` these formula!

**Comment by kennethreitz** [comment]
Deployed to `cedar-14`.

**Comment by kennethreitz** [comment]
Deployed to `heroku-16`.

**Comment by joshfriend** [comment]
> For next time: make sure you chmod +x these formula!

Thank you, I'll try to remember that!

**Pull request merged**